recipes/pcaone/meta.yaml

```diff
@@ -17,19 +17,21 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make
     - sysroot_linux-64 2.17  # [linux]
   host:
-    - llvm-openmp  # [osx]
-    - libgomp  # [linux]
-    - mkl
-    - mkl-include
+    - mkl >=2025.1.0  # [linux]
+    - mkl-include >=2025.1.0  # [linux]
     - zlib
+    - llvm-openmp # [osx]
+    - libgomp # [linux]
   run:
-    - mkl
-    - llvm-openmp  # [osx]
-    - libgomp  # [linux]
+    - mkl >=2025.1.0  # [linux]
+    - mkl-include >=2025.1.0  # [linux]
```

Comment:
mkl is not needed at all on macos? just checking.

Reply:
Ok... this is weird. It told you to do that because you have the 0.5.2 here, but the latest in the repo is 0.4.8. Somehow, the meta.yaml must have gotten updated without new packages entering the repo?

Leave it at 0 then. That's correct. It's just odd that no previous 0.5.2 exists...